### DISCUSSION TITLE
How to run Prisma with Load Balancer?

### DISCUSSION THREAD

**Discussion by junsik5477** [prisma/prisma#3033] [Q&A]
versions
 - prisma/cli: 2.2.2
 - prisma/client:2.2.1

Looking for a way to have Prisma on Multiple Hosts for a MariaDB . 
The Prisma  will have a Load Balancer with 5 Hosts.

Is it Possible ? if Yes, Please suggest steps ?

**Comment by ryands17** [comment]
Hey @junsik5477 👋 
Are you looking to run multiple databases and connect to them via Prisma? Could you explain your use case in a bit more detail?
